botCommands/carlosQuote.js

```diff
@@ -1,17 +1,27 @@
+const Discord = require('discord.js');
 const { registerBotCommand } = require('../botEngine');
 
 const command = {
   regex: /(?<!\S)!cqg(?!\S)/,
-  cb: () => `**Here is a great quote about "Asking Questions in the TOP Discord"**
-    "Here's my take on asking questions here:
-    Naturally, we welcome them. But we will hold you accountable when you ask a question that is easily researched. Not because we think you're a jerk. But because independent research is so critical to your continued learning.
-    Imagine you are on your first job and you went to your lead dev for questions every few minutes. I'm sure they'd be happy to help but a better situation is if you do some initial research on your own then take what you've found to your lead if you still have questions.
-    That's a very different conversation from: "Hey, I have no idea about this. Please tell me everything."
-    vs
-    "So I did some research. This is what I understand. And this is what I don't understand. Can you help clarify this last bit?"
-    So yeah, feel free to continue to ask. But make sure you are doing your part by doing the research. This helps all of us grow."
-    **-Carlos**`,
+  cb: () => {
+    const template = new Discord.EmbedBuilder()
+      .setColor('#cc9543')
+      .setTitle('Asking Questions in the TOP Discord')
+      .addFields([
+        {
+          name: '\u200B',
+          value:
+            "\"Here's my take on asking questions here:\n\nNaturally, we welcome them. But we will hold you accountable when you ask a question that is easily researched. Not because we think you're a jerk. But because independent research is so critical to your continued learning.\n    \nImagine you are on your first job and you went to your lead dev for questions every few minutes. I'm sure they'd be happy to help but a better situation is if you do some initial research on your own then take what you've found to your lead if you still have questions.\n    \nThat's a very different conversation from: *\"Hey, I have no idea about this. Please tell me everything.\"*\n\nvs\n\n*\"So I did some research. This is what I understand. And this is what I don't understand. Can you help clarify this last bit?\"*\n\nSo yeah, feel free to continue to ask. But make sure you are doing your part by doing the research. This helps all of us grow.\"",
+        },
+      ])
```

Comment:
Instead of a field with empty `name` field, I think it makes sense to just use `.setDescription` function here. You can find examples of `.setDescription` in other commands.

Reply:
Working on it. I was kind of learning as I went. I used the `.setDescription` in a newer PR I think. Great call and fixing it now!